### DISCUSSION TITLE
boxedwine automatically stops when installer closes

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#239] [Feature Requests]
when I am trying to use something like chrome or steam on the boxedwine emulator, the emulator stops when the installer has finished. has this been added to daedalOS or is it built into the emulator?

**Comment by DustinBrett** [comment]
BoxedWine mostly does it's own thing and I have an outdated version. I plan to update in the near future once they move to Wine 7 (https://github.com/danoon2/Boxedwine/issues/63#issuecomment-1318941986). Until then afaik the `exe` you load initially will crash BoxedWine if it closes.
